### ISSUE TITLE
Code coverage

### ISSUE THREAD

**Issue by gjhuizing** [ott-jax/ott#254]
Some files bring down the test coverage,

- src/ott/geometry/graph.py
- src/ott/tools/plot.py
- src/ott/solvers/nn/neuraldual.py
- src/ott/math/decomposition.py
- src/ott/problems/linear/potentials.py
- src/ott/types.py
- src/ott/problems/quadratic/quadratic_costs.py

Maybe more tests could be created for those?

Source: https://app.codecov.io/gh/ott-jax/ott?displayType=list

**Comment by michalk8** [comment]
@bamos in preparations to focus on this issue, could you please create a PR similar to https://github.com/MUCDK/ott-1/pull/1 to only fix `amortization_loss='objective'` issue + add the parameters to the test? #289 will take longer to merge as @MUCDK is on vacation.

**Comment by bamos** [comment]
> @bamos in preparations to focus on this issue, could you please create a PR similar to [MUCDK#1](https://github.com/MUCDK/ott-1/pull/1) to only fix `amortization_loss='objective'` issue + add the parameters to the test? #289 will take longer to merge as @MUCDK is on vacation.

Yes, I just sent that in: https://github.com/ott-jax/ott/pull/327

I also removed the initial parameters for now since I'm not using them, but in the future it could be useful to add support for suspending/resuming training runs, which would also need to take the training state and other iteration/logging info.
